assets/aml-benchmark/scripts/custom_inference_postprocessors/humaneval.py

```diff
@@ -184,17 +184,24 @@ def run_humaneval_postprocessor(
     # Post processing the prediction and ground truth columns
     for row in pred_dict_full:
         gt = "\n" + row["test"] + "\n" + "check(" + row["entry_point"] + ")"
-        if str("def " + row["entry_point"] + "(") in row["original_prediction"]:
-            # If the model regenerates the prompt/ function name
-            pred_combined_prompt = _extract_text_from_markdown_tag(row["original_prediction"], tag_type='python')
-        else:
-            original_prediction = _extract_text_from_markdown_tag(row["original_prediction"], tag_type='python')
+        tag_type = "python" if "python" in row["original_prediction"] else ""
```

Comment:
Can this condition be-
```
tag_type = "python" if "```python" in row["original_prediction"] else ""
```
This will handle the scenario when the model prediction is something like (where python is not in ```-
```
Sure, here is the code in python: ```\nreturn arr.sort()```.
```